main.virtualnetwork.tf

```
@@ -6,6 +6,7 @@ module "virtualnetwork" {
   subscription_id  = local.subscription_id
   virtual_networks = var.virtual_networks
   location         = var.location
+  enable_telemetry = var.disable_telemetry == false ? true : false
```

could this be simplified?

```suggestion
  enable_telemetry = !var.disable_telemetry
```